History.markdown

```diff
@@ -7,6 +7,7 @@
 ### Minor Enhancements
 
   * Excerpt only flag (#287)
+  * Add media:content tag
```

Comment:
We add this with @jekyllbot. Would you please remove?

Reply:
Done